initrd/bin/install.init

```diff
@@ -20,7 +20,7 @@ echo ''
 echo 'To install from flash drive:'
 echo ''
 echo '  mount -o ro /dev/sdb1 /media'
-echo '  flashrom-x230.sh /media/x230.rom'
+echo '  flash.sh /media/<rom-file-name>'
 echo ''
```

Comment:
has already been fixed...

Reply:
it wasn't back when those changes were actually made.

Reply:
> has already been fixed...

@merge, where?

Reply:
in https://github.com/osresearch/heads/commit/00bf4740e3c08feb1870f4c7b4dd7823681bf22c